test/grapple_test.exs

```diff
@@ -46,7 +54,7 @@ defmodule GrappleTest do
 
     test "sends a successful hook", %{hook: hook} do
       Hook.subscribe hook
-      [resp] = Hook.broadcast hook.topic
+      [%{response: resp}] = Hook.broadcast hook.topic
```

Comment:
We should also update the README as well since the assignment pattern here has changed